modules/core/src/opengl.cpp

```diff
@@ -1638,14 +1638,14 @@ Context& initializeContextFromGL()
     cl_uint numPlatforms;
     cl_int status = clGetPlatformIDs(0, NULL, &numPlatforms);
     if (status != CL_SUCCESS)
-        CV_Error(cv::Error::OpenCLInitError, "OpenCL: Can't get number of platforms");
+        CV_Error_(cv::Error::OpenCLInitError, "OpenCL: Can't get number of platforms: %d ", status);
```

Comment:
Is it tested anywhere on CI?

Used syntax is following:

`CV_Error_(core, (format, args...))`

Reply:
I got your point and fix it asap.

Reply:
I am testing at the moment. Why did this work?

Reply:
A merge gone bad🤦‍♂️
Still testing.